server/context/conf.js

```diff
@@ -85,17 +85,22 @@ if (IS_COVERAGE_ACTIVE) {
   }
 
   // Source maps remapping
+  let defFormats = ['html', 'cobertura', 'clover', 'json', 'json-summary', 'lcovonly', 'teamcity', 'text', 'text-summary'];
```

Comment:
I would prefer to store these in the `/conf/default-coverage.json` file to remove these constants from the `Conf.js` file